src/Nri/Ui/SlideModal/V2.elm

```diff
@@ -257,10 +259,28 @@ viewContent content =
     Nri.Ui.styled div
         "modal-content"
         [ Css.overflowY Css.auto
+        , Css.margin2 Css.zero (Css.px 21)
         , Css.padding2 (Css.px 30) (Css.px 40)
         , Css.width (Css.pct 100)
+        , Css.height (Css.pct 100)
         , Css.marginBottom Css.auto
         , Css.boxSizing Css.borderBox
+
+        -- Shadows for indicating that the content is scrollable
+        , Css.property "background"
+            """
+            /* TOP shadow */
+
+            top linear-gradient(to top, rgb(255, 255, 255), rgb(255, 255, 255)) local,
+            top linear-gradient(to top, rgba(255, 255, 255, 0), rgba(0, 0, 0, 0.15)) scroll,
```

Comment:
ohhhhh, this is super clever! I would have thought we would have needed some scripting/interaction to make this work well. 😻

Reply:
It was google's idea! I was pretty excited :)